core/src/main/java/nl/nn/adapterframework/webcontrol/api/ShowMonitors.java

```diff
@@ -163,7 +163,7 @@ public Response getMonitors(@PathParam("configuration") String configurationName
 	private Map<String, Object> mapTrigger(ITrigger trigger) {
 		Map<String, Object> triggerMap = new HashMap<String, Object>();
 
-		triggerMap.put("type", trigger.getType());
+		triggerMap.put("type", trigger.getTypeDisplayString());
```

Comment:
Hier moet hij misc.initialCaps(enum.name()) kunnen doen. Met de extra toLowerCase in de initialCaps methode

Reply:
Hopelijk gedaan. Is dit wat je bedoelt?